components/Navbar/Navbar.js

```diff
@@ -98,6 +105,11 @@ export default function Navbar() {
       <Modal open={modalOpen} dismissible onClose={() => setModalOpen(false)}>
         <EditBlogForm onSuccess={() => setModalOpen(false)} />
       </Modal>
+      {!user && (
+        <Modal open={showLoginModal} dismissible onClose={() => setShowLoginModal()}>
+          <Login onSuccess={() => setShowLoginModal()} />
```

Comment:
missing param for setShowLoginModal

Reply:
you would still need a true or false param since `setShowLoginModal` comes from useState. 

the modal closes after signin is because the state of `!user` changed